code/modules/mob/living/basic/space_fauna/paper_wizard/paper_wizard.dm

```diff
@@ -0,0 +1,187 @@
+/mob/living/basic/paper_wizard
+	name = "Mjor the Creative"
+	desc = "A wizard with a taste for the arts."
+	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	faction = list(FACTION_HOSTILE, FACTION_STICKMAN)
+	icon = 'icons/mob/simple/simple_human.dmi'
+	icon_state = "paperwizard"
+	gender = MALE
+
+	response_help_continuous = "brushes"
+	response_help_simple = "brush"
+	response_disarm_continuous = "pushes"
+	response_disarm_simple = "push"
+
+	maxHealth = 1000
+	health = 1000
+	melee_damage_lower = 10
+	melee_damage_upper = 20
+	attack_sound = 'sound/hallucinations/growl1.ogg'
+	var/list/copies = list()
+	var/datum/action/cooldown/spell/wizard_summon_minions/summon
+	var/datum/action/cooldown/spell/pointed/wizard_mimic/mimic
+
+	ai_controller = /datum/ai_controller/basic_controller/paper_wizard
```

Comment:
old vars before new vars, `ai_controller` should go before all of these. Would also be nice if you could dmdoc the three new vars for statistical purposes.